torch/_inductor/lowering.py

```diff
@@ -2490,12 +2490,15 @@ def apply_constraint(idx, arg, fx_arg):
             out_size = list(arg.get_size())
 
             expanded_dims = []
-            if arg.maybe_get_stride() is not None:
-                # We require a dense last dimension, but the other strides
-                # can be expanded, which results in a smaller tensor
-                for i, s in enumerate(arg.get_stride()[0:-1]):
-                    if V.graph.sizevars.statically_known_equals(s, 0):
-                        expanded_dims.append(i)
+            # We require a dense last dimension, but the other strides
+            # can be expanded, which results in a smaller tensor
+            maybe_stride = arg.maybe_get_stride()
```

Comment:
Should the bias arg already be a ExpandView here ? (which is what we want to return).

Reply:
yea, maybe this can all be simplifier 🤔 let me try to update